plottable.d.ts

```diff
@@ -3577,6 +3577,93 @@ declare module Plottable {
 
 
 declare module Plottable {
+    module Plots {
+        class Wheel<R, T> extends Plot {
+            /**
+             * @constructor
+             */
+            constructor();
+            computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number): Wheel<R, T>;
+            protected _createDrawer(dataset: Dataset): Drawers.Arc;
+            entities(datasets?: Dataset[]): PlotEntity[];
+            protected _getDataToDraw(): Utils.Map<Dataset, any[]>;
+            protected _propertyProjectors(): AttributeToProjector;
+            /**
+             * Gets the AccessorScaleBinding for the start angle.
+             */
+            startAngle<T>(): AccessorScaleBinding<T, number>;
+            /**
+             * Sets the start angle to a constant number or the result of an Accessor<number>.
+             *
+             * @param {number|Accessor<number>} startAngle
+             * @returns {Wheel} The calling Wheel Plot.
+             */
+            startAngle(startAngle: number | Accessor<number>): Plots.Wheel<R, T>;
+            /**
+             * Sets the start angle to a scaled constant value or scaled result of an Accessor.
```

Comment:
Currently this method expects radians, but I wonder if we should expect degrees instead? `angle()` on `Label` and `tickLabelAngle()` on `Axes.Category` take degrees, for example (but not all possible values). Maybe worth an RFC? Or follow up on the existing thread.

Either way, we should specify the expected units in the docs.